portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -52,15 +53,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String commentAuthor = getNickname(id);
       long commentTimeMS = (long) entity.getProperty("time");
       Date commentTime = new Date(commentTimeMS);
+      double commentToxicity = (double) entity.getProperty("toxicity");
 
-      Comment comment = new Comment(commentText, commentAuthor, commentTime);
-      comments.add(comment);
+      if (commentToxicity <= maxToxicity) {
+        Comment comment = new Comment(commentText, commentAuthor, commentTime, commentToxicity);
+        comments.add(comment);
+      }
 
       if (comments.size() == maxComments) {
         break;
       }
     }
 
+    // Edge case: all comments are greater than toxicity tolerance
+    if (comments.size() == 0 && results != null) {
+      Comment comment = new Comment("There are no comments satisfying your search request.", "Comment Bot", new Date(), 0);
+      comments.add(comment);
```

Comment:
Curious, would it be possible to send just the empty list and then handle this condition on the client-side? It feels a little odd to send a `Comment` to indicate there are no results.

Reply:
This was for debugging - I'll change it now!